src/app/frontend/chrome/template.html

```diff
@@ -54,7 +54,8 @@
   <mat-toolbar class="kd-second-toolbar kd-bg-primary">
     <div class="kd-toolbar-tools"
          fxFlex>
-      <kd-nav-hamburger fxFlex="none"></kd-nav-hamburger>
+      <kd-nav-hamburger *ngIf="showNav()"
```

Comment:
We should use nav service for that. This feels more like a workaround. If nav service does not work, then just leave it visible for now (restore previous fix).